src/components/projects/new/edit-activity.svelte

```diff
@@ -63,11 +63,11 @@
   }
 
   function validateAndSave() {
-    if (!activity.name) {
+    if (!activity.name || /^\s*$/.test(activity.name)) {
```

Comment:
Regex is slower than native methods
```suggestion
    if (!activity.name.trim()) {
```